client/.storybook/main.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  stories: ["../src/**/*.stories.js"],
+  addons: [
+    "@storybook/preset-create-react-app",
+    "@storybook/addon-actions",
+    "@storybook/addon-links",
+  ],
```

Comment:
When you install Knobs you need to add `"@storybook/addon-knobs/register"`
https://www.npmjs.com/package/@storybook/addon-knobs

Reply:
thank you very much!